cluster-scope/overlays/ocp-staging/kustomization.yaml

```diff
@@ -3,3 +3,21 @@ kind: Kustomization
 resources:
   # please keep these items sorted
   - ../../bundles/external-secrets
+  - ../../bundles/cert-manager
+  - apiservers/cluster.yaml
+  - certificates/api-certificate-letsencrypt.yaml
+  - certificates/default-ingress-certificate.yaml
+
+patches:
+  - path: clusterissuers/patch.yaml
+    target:
+      kind: ClusterIssuer
+      name: letsencrypt-production-dns01
+  - path: clusterissuers/patch.yaml
```

Comment:
We don't need either of the namespaced issuers, since we're going to be using the clusterissuer.

Reply:
okay, yeah they were in base but I didn't want them anything to do with ocp-prod so I patched it, but now I learnt I can just delete things from base in a patch.

Reply:
I think we should just probably land a pr that removes the namespaced issuers first, rather than patching them out. I can put that together tomorrow.

Reply:
Like https://github.com/CCI-MOC/moc-apps/pull/77

Reply:
okay, rebased and updated